test/powershell/engine/Remoting/RemoteSession.Basic.Tests.ps1

```diff
@@ -8,4 +8,23 @@ Describe "New-PSSession basic test" -Tag @("CI") {
             $_.FullyQualifiedErrorId | Should Be "InvalidOperation,Microsoft.PowerShell.Commands.NewPSSessionCommand"
         }
     }
+}
+
+Describe "JEA session bug fix test" -Tag @("Feature", 'RequireAdminOnWindows') {
```

Comment:
the test  suite contains different bug fixes for JEA sessions. The information is contained in each test itself. thank you.